app/assets/stylesheets/components/_table.scss

```diff
@@ -0,0 +1,20 @@
+table.dataTable {
+  border: 1px solid lighten($light-gray, 25%);
+  box-shadow: none;
+  width: 100% !important;
+
+  thead, tfoot, tr, tr.even { background: white; }
+  tr td,
```

Comment:
Selector should have depth of applicability no greater than 2, but was 4<br>Each selector in a comma sequence should be on its own line